integration/grpc.go

```diff
@@ -0,0 +1,29 @@
+package integration
+
+import (
+	"context"
+	"fmt"
+
+	jwt "github.com/dgrijalva/jwt-go"
+	"google.golang.org/grpc/metadata"
+)
+
+// AppendTokenToOutgoingContext adds an authorization token to the gRPC
+// request context metadata. The user must provide a token field name like "token"
+// or "bearer" to this function. It is intended specifically for gRPC testing.
```

Comment:
it would be good to add extra info here on example usage, or create a testable example demonstrating how to use this function

Reply:
Yeah, I like the idea of creating a testable example.